move/interchain_token_service/sources/events.move

```diff
@@ -73,12 +73,21 @@ module interchain_token_service::events {
         new_operator: Option<address>,
     }
 
-    public struct InterchainTokenIdClaimed<phantom T> has copy, drop {
+    public struct InterchainTokenIdClaimed has copy, drop {
```

Comment:
Modifying an existing struct/type won't work?

Reply:
it would since this is added after the upgrade, but re-added nevertheless